diff --git a/src/Store/LemonSqueezyApi.php b/src/Store/LemonSqueezyApi.php
index d16edfc..5c6cd66 100644
--- a/src/Store/LemonSqueezyApi.php
+++ b/src/Store/LemonSqueezyApi.php
@@ -87,8 +87,7 @@ final readonly class LemonSqueezyApi
 
     public function retrieveStoreUrl(): string
     {
-        $response = $this->client->request(Request::METHOD_GET, 'stores/' . $this->storeId);
-        $lsStore = $response->toArray();
+        $lsStore = $this->request(Request::METHOD_GET, 'stores/' . $this->storeId);
 
         return $lsStore['data']['attributes']['url'];
     }
@@ -100,7 +99,8 @@ final readonly class LemonSqueezyApi
         }
 
         $lsCustomer = $this->retrieveCustomer($user->getLsCustomerId());
-        $response = $this->client->request(Request::METHOD_GET, 'orders', [
+
+        return $this->request(Request::METHOD_GET, 'orders', [
             'query' => [
                 'filter' => [
                     'store_id' => $this->storeId,
@@ -111,15 +111,11 @@ final readonly class LemonSqueezyApi
                 ],
             ],
         ]);
-
-        return $response->toArray();
     }
 
     public function retrieveCustomer(string $customerId): array
     {
-        $response = $this->client->request(Request::METHOD_GET, 'customers/' . $customerId);
-
-        return $response->toArray();
+        return $this->request(Request::METHOD_GET, 'customers/' . $customerId);
     }
 
     private function request(string $method, string $url, array $options = []): array
