diff --git a/src/Store/LemonSqueezyApi.php b/src/Store/LemonSqueezyApi.php
index 13724d5..9a2d88b 100644
--- a/src/Store/LemonSqueezyApi.php
+++ b/src/Store/LemonSqueezyApi.php
@@ -132,7 +132,7 @@ final readonly class LemonSqueezyApi
             $data = $response->toArray();
         } catch (ClientException $e) {
             $data = $e->getResponse()->toArray(false);
-            dd($data);
+            //dd($data);
         }
 
         return $data;
