diff --git a/src/Store/LemonSqueezyApi.php b/src/Store/LemonSqueezyApi.php
index 024e829..b693f3a 100644
--- a/src/Store/LemonSqueezyApi.php
+++ b/src/Store/LemonSqueezyApi.php
@@ -131,7 +131,7 @@ final readonly class LemonSqueezyApi
             $data = $response->toArray();
         } catch (ClientException $e) {
             $data = $e->getResponse()->toArray(false);
-            dd($data);
+            //dd($data);
         }
 
         return $data;
